tests/conftest.py

```diff
@@ -320,7 +320,7 @@
 
 
 @contextmanager
-def project_root_pythonpath() -> None:
```

Comment:
In the base PR I made an mistake of annotating these functions since the mypy was complaining about none defined return types which was caused because of some of package tests imported stuff from `tests.conftests` and mypy also scanned `tests.conftests` which is not the case now. So I've reverted the changes